readme.md

```diff
@@ -11,21 +11,11 @@ It returns the latest repos along with some metadata. The result is cached for a
 
 ## Usage
 
-### With [`now`](https://now.sh)
-
-[![Deploy to now](https://deploy.now.sh/static/button.svg)](https://deploy.now.sh/?repo=https://github.com/sindresorhus/gh-latest-repos&env=GITHUB_TOKEN&env=GITHUB_USERNAME&env=ACCESS_ALLOW_ORIGIN&env=MAX_REPOS&env=CACHE_MAX_AGE)
```

Comment:
Why remove the badge?

Reply:
It doesn't work anymore.

Reply:
Wow. I always thought it was an official thing because of the subdomain `https://deploy.now.sh`.